__tests__/dependencies.ts

```diff
@@ -20,7 +20,7 @@ describe(
                     mapperService: expect.any(Function)
                 });
                 const depsDb = dependencies(['dbClient']);
-                await depsDb.dbClient.close();
+                await depsDb!.dbClient!.close();
```

Comment:
interesting, I do not have to use the non-null assertion. Maybe I do not have the same typescript setup.